src/onig-scanner-worker.cc

```diff
@@ -15,17 +15,17 @@ void OnigScannerWorker::Execute() {
 }
 
 void OnigScannerWorker::HandleOKCallback() {
-  NanScope();
+  Nan::HandleScope();
```

Comment:
We need to give it a name since `Nan::HandleScope` is a class and no longer a macro:

```c++
Nan::HandleScope scope;
```